tensorflow/core/platform/s3/s3_file_system.cc

```diff
@@ -240,6 +285,8 @@ S3FileSystem::S3FileSystem() {
 S3FileSystem::~S3FileSystem() {
   Aws::SDKOptions options;
   Aws::ShutdownAPI(options);
+
+  Aws::Utils::Logging::ShutdownAWSLogging();
```

Comment:
Shouldn't we be shutting down the `kS3FileSystemLoggingTag` logging only?

Reply:
Is there a way to fix this?

Reply:
Thanks @drpngx for the review. The `kS3FileSystemLoggingTag` is the tag for creating memory when invoking the logging. As it is not to create a per-tag logging so the shutdown of logging does not have a parameter.

Reply:
OK, it doesn't looks like there is. Also, the ShutdownAPI is a global call of similar nature. That's not great.

Could we have the InitializeAWSLogging and ShutdownAWSLogging exposed as static functions in S3Logging? At least we can control behavior from there. If there are multiple users, we could ref count them so that the shutdown is called only once globally.

Reply:
@drpngx Thanks. Done.